docs/user-guide/03_technical/01_prefida_inputs.md

```diff
@@ -332,5 +332,5 @@ The variable x is a plasma parameter defined by `adaptive` according to the foll
 |       Variable      |   Type  | Rank |  Dimensions  | Units |           Description           |
 |:-------------------:|:-------:|:----:|:------------:|:-----:|:--------------------------------| 
 |`adaptive`           |Int32    |0     |NA            |NA     |Calculate `n_cells` according to plasma parameter|
-|`split_tol`          |Float64  |0     |NA            |NA     |Split tolerance, %/cm            |
+|`split_tol`          |Float64  |0     |NA            |cm^-1  |Split tolerance, %/cm            |
```

Comment:
technically not a percentage. We don't want using putting a value of 10 when they meant 0.1